.github/workflows/sdk-client-ci.yml

```diff
@@ -34,15 +37,17 @@ jobs:
 
       - uses: ./.github/actions/ci
         with:
-          project_file: pkgs/sdk/client/src/LaunchDarkly.ClientSdk.csproj
-          test_project_file: pkgs/sdk/client/test/LaunchDarkly.ClientSdk.Tests/LaunchDarkly.ClientSdk.Tests.csproj
+          project_file: ${{ env.PROJECT_FILE}}
+          test_project_file: ${{ env.TEST_PROJECT_FILE}}
 
       - uses: ./.github/actions/contract-tests
         with:
-          service_project_file: pkgs/sdk/client/contract-tests/TestService.csproj
-          service_dll_file: pkgs/sdk/client/contract-tests/bin/debug/net7.0/ContractTestService.dll
+          service_project_file: ${{ env.CONTRACT_TEST_PROJECT_FILE}}
+          service_dll_file: ${{ env.CONTRACT_TEST_DLL_FILE}}
           token: ${{ secrets.GITHUB_TOKEN }}
 
       - uses: ./.github/actions/build-docs
         with:
-          workspace_path: pkgs/sdk/client
+          workspace_path: ${{ env.WORKSPACE_PATH}}
+
+# TODO: resolve issue with emulator and device tests
```

Comment:
For reviewers: This is an existing TODO in the client-sdk repo that I am migrating here so we don't forget about it.